docs/_docs/2_6_deployment.md

```diff
@@ -225,41 +238,128 @@ hudi:trips->connect --path /app/uber/trips
 Metadata for table trips loaded
 ```
 
-Once connected to the table, a lot of other commands become available. The shell has contextual autocomplete help (press TAB) and below is a list of all commands, few of which are reviewed in this section
-are reviewed
+Once connected to the table, a lot of other commands become available. The shell has contextual autocomplete help (press TAB) and below is a list of all commands, few of which are reviewed in this section.
+
+ - `commits show` - Lists all commits in hudi table.
+ - `commits showarchived` - Lists all archived commits in hudi table.
+ - `commits refresh` - Refreshes the commit timeline. If there are ongoing operations after launching hudi-cli tool, this will refresh your commits to include all latest commits too.
+ - `commits compare` - Compares commits with another hudi table.
+ - `commits sync` - Syncs current hudi table with another table.
+ - `commit show_write_stats` - Lists write stats for commit of interest. A commit id need to be provided as an argument, for example
+
+```java
+commit show_write_stats --commit 20210501233331
+```
+
+ - `commit showfiles` - Lists files for a given commit.
+ - `commit showpartitions` - Shows partition information for a given commit.
+ - `commit rollback` - Rolls back a commit.
+
+```java
+commit rollback --commit 20210501233331
+```
+
+ - `stats filesizes`: Display summary stats on sizes of files
```

Comment:
can we please add this example too 
stats filesizes --partitionPath <PartitionPath>

Reply:
It is already present here - https://hudi.apache.org/docs/deployment.html#statistics